src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Completion/AutoCompletionResult.cs

```diff
@@ -0,0 +1,68 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.SqlServer.Management.SqlParser.Intellisense;
+using Microsoft.SqlTools.ServiceLayer.LanguageServices.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.LanguageServices.Completion
+{
+    /// <summary>
+    /// Includes the objects created by auto completion service
+    /// </summary>
+    public class AutoCompletionResult
+    {
+        /// <summary>
+        /// Creates new instance
+        /// </summary>
+        public AutoCompletionResult()
+        {
+            Stopwatch = new Stopwatch();
+            StartProcesssing();
+        }
+
+        /// <summary>
+        /// Starts processing the result
+        /// </summary>
+        public void StartProcesssing()
```

Comment:
>public [](start = 8, length = 6)

I couldn't see this method being used outside the class (may have missed it though)..if not used outside you could make private or just move Stopwatch.Start into constructor directly.